LOGGING/test/Log4NetAppender.Net45.Tests/ApplicationInsightsAppenderTests.cs

```diff
@@ -325,6 +325,7 @@ public void CustomMessageIsAddedToExceptionTelemetryCustomProperties()
             }
 
             ExceptionTelemetry telemetry = (ExceptionTelemetry)this.appendableLogger.SentItems.First();
+            Assert.IsTrue(telemetry.Message.Contains("Test logging exception"));
```

Comment:
the Message contains not just the "Test logging exception" but the exception type as well right? Can you validate taht as well in tests

Reply:
Sure